controllers/profilecatalogsource_controller.go

```diff
@@ -41,6 +41,10 @@ type ProfileCatalogSourceReconciler struct {
 // +kubebuilder:rbac:groups=weave.works,resources=profilecatalogsources/status,verbs=get;update;patch
 // +kubebuilder:rbac:groups=weave.works,resources=profilecatalogsources/finalizers,verbs=update
 
+// Reconcile is part of the main kubernetes reconciliation loop which aims to
+// move the current state of the cluster closer to the desired state.
+// For more details, check Reconcile and its Result here:
+// - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.7.0/pkg/reconcile
 func (r *ProfileCatalogSourceReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
```

Comment:
how did this pass the linter before :thinking:

Reply:
I have no idea but I say it a couple of times now and it was just annoying me at this point. :D